aion/lib/aion/room_channel/room.ex

```diff
@@ -51,7 +58,13 @@ defmodule Aion.RoomChannel.Room do
   @spec change_question(%__MODULE__{}, integer) :: __MODULE__.t
   def change_question(room, room_id) do
     room
-    |> struct(get_new_question_with_answers(room_id))
+    |> struct(get_new_question_with_answers(room.questions, room_id))
+  end
+
+  @spec load_questions(%__MODULE__{}, integer) :: __MODULE__.t
+  def load_questions(room, room_id) do
+    room
+    |> struct(get_new_questions(room_id))
```

Comment:
Mixing a bunch of parentheses and pipeline approach doesn't look good. Since the pipe is not long here, you could always just go with 
```
questions = get...
struct(room, questions)
```